### PR TITLE
Row row hero cleanup

### PR DESCRIPTION
**Pull request by pmahnke**
## Demo

http://www.ubuntu.com-row-row-hero-cleanup.demo.haus/

[spreadsheet where I calculated common patterns](https://docs.google.com/a/canonical.com/spreadsheets/d/1ccyunLOkW2ULHUBTOaEb3PG9_bG8cr2BF9a8TT8F4d8/edit?usp=sharing)
## Done
- on /desktop/enterprise, removed padding-bottom override on 'Enterprise-wide' section
- /desktop/education, removed padding-bottom override on 'Support and management' section
- on /desktop/developers, removed padding-bottom override on 'Support tailored' section
- on /desktop/government, removed padding-bottom override on 'Government-wide support ' section
- added back some margin-left on non-full-width breadcrumb links
- on /internet-of-things removed h1 padding-top for the row-hero
## Todo
- remove row and row-hero overrides in _download.scss when my PR is approved
- figure out how to fix / - homepage takeunders?  padding: 100px 40px;
  margin: -100px 0 -60px 0; ???? medium looks messed up still...
## Questions
- on /internet-of-things/digital-signage - could not use equal height on row-hero?
- on /internet-of-things/robotics - could not use equal height on row-hero?

### CODE REVIEW

**Comment by pmahnke** [comment]
[review row/row-hero pr](https://trello.com/c/iFjFpGNQ/2050-review-row-row-hero-pr)

**Comment by yaili** [comment]
Had a look, here are my comments:

- [ ] /cloud/maas: hero image not positioned correctly
- [ ] /cloud/partners: hero image not positioned correctly and it’s making a strange effect on page load
- [ ] /desktop/government: text not in same place as other pages
- [ ] /phone/features: text not in same place as other pages
- [ ] download section has text higher than rest of site - this is ok, just noting it
- [ ] /download/server: third level pages not highlighted in nav

**Comment by deadlight** [comment]
@pmahnke try rebasing this one, I'm not sure that the conflicts are anything that bad. If you want me to have a go at fixing it, let me know.

**Comment by pmahnke** [comment]
@deadlight ready for review

File: static/css/section/_cloud.scss [hunk 1]
```diff
@@ -496,6 +490,51 @@
     }
   }
 
+  // @subsection /cloud/public-cloud
+  &.cloud-public-cloud {
```
Comment (nottrobin) [inline static/css/section/_cloud.scss]: I can't find any mention of `class="cloud-public-cloud"` in the codebase - do you need to add a whole lot of new styling for it?

File: static/css/section/_things.scss [hunk 2]
```diff
@@ -0,0 +1,314 @@
+@charset 'UTF-8';
```
Comment (nottrobin) [inline static/css/section/_things.scss]: Are you intentionally adding this file? Was this a bad merge? I can't find mention if it being included anywhere, or of the `internet-of-things` class being used.

File: templates/download/server/arm.html [hunk 3]
```diff
@@ -4,15 +4,15 @@
...
+        {% include "templates/_nav_breadcrumb.html" with section_title="Download" subsection_title="Server" page_title="ARM" %}
     </div>
 {% endblock second_level_nav_items %}
 
 {% block content %}
 <div class="row row-grey">
     <div class="strip-inner-wrapper">
         <div class="eight-col">
-            <h1>Ubuntu Server for ARM</h1>
+            <h1>Ubuntu Server for ARM {{level_1}} / {{level_2}} / {{level_3}} / {{level_4}}</h1>
```
Comment (nottrobin) [inline templates/download/server/arm.html]: I can't imagine this was intentional...

File: static/css/section/_cloud.scss [hunk 4]
```diff
@@ -178,15 +178,15 @@
     .row-maas {
 
       @media only screen and (min-width : $breakpoint-medium) {
-        background-image: url('#{$asset-server}1066e824-cloud_maas.jpg?op=region&rect=0,0,300,500');
-        background-position: right top;
-        background-repeat: no-repeat;
+        //background-image: url('#{$asset-server}1066e824-cloud_maas.jpg?op=region&rect=0,0,300,500');
```
Comment (nottrobin) [inline static/css/section/_cloud.scss]: Could you remove these commented-out lines? Or add a note saying why you're leaving them in?

File: static/css/section/_cloud.scss (hunk 1 above)
Comment (pmahnke) [inline static/css/section/_cloud.scss]: that's a body class that is constructed

image

**Comment by pmahnke** [comment]
@nottrobin can you look again?

File: static/css/section/_cloud.scss (hunk 1 above)
Comment (nottrobin) [inline static/css/section/_cloud.scss]: A constructed class? Yuck! Okay fair enough.

**Pull request merged**